[DOCS] Remove examples of specifying a test_backends list of dicts (#7816)
great-expectations/great_expectations: docs/docusaurus/docs/guides/expectations/creating_custom_expectations/how_to_create_custom_batch_expectations.md

	* `in` contains exactly the parameters that you want to pass in to the Expectation. `"in": {"strict": True}` in the example above is equivalent to `expect_batch_columns_to_be_unique(strict=True)`
	* `out` is based on the Validation Result returned when executing the Expectation.
	* `exact_match_out`: if you set `exact_match_out=False`, then you don’t need to include all the elements of the Validation Result object - only the ones that are important to test.

> * `only_for` (optional): list of backends that the Expectation should work with

Reviewer: ```suggestion
* `only_for` (optional): the list of backends that the Expectation should use for testing
```

> * `suppress_test_for` (optional): list of backends that the Expectation does not work with

Reviewer: ```suggestion
* `suppress_test_for` (optional): the list of backends that the Expectation should not use for testing
```

> * `only_for` and `suppres_test_for` may be specified at the top-level (next to `data` and `tests`) or within specific tests (next to `title`, etc)

Reviewer: ```suggestion
* `only_for` and `suppres_test_for` can be specified at the top-level (next to `data` and `tests`) or within specific tests (next to `title`, and so on)
```
![image](https://user-images.githubusercontent.com/104205257/236830606-530447a2-25d7-4cce-b260-0d598f2b8631.png)
